lib/pluginmanager/update.rb

```diff
@@ -6,6 +6,7 @@ class LogStash::PluginManager::Update < LogStash::PluginManager::Command
   REJECTED_OPTIONS = [:path, :git, :github]
 
   parameter "[PLUGIN] ...", "Plugin name(s) to upgrade to latest version", :attribute_name => :plugins_arg
+  option "--[no-]verify", :flag, "verify plugin validity before installation", :default => true
```

Comment:
Yes! because we introduced a feature to warn for major versions. Please take a look at the code and you will understand. 

The code used only talks to rubygems.org and not to others private repositories that might not share the same api.